### ISSUE TITLE
Always showing "insufficient space"

### ISSUE THREAD

**Issue by th3d00d** [cbepx-me/Anbernic-H700-RG-xx-StockOS-Modification#37]
Windows is always showing "insufficient space" when trying to copy the upgrade files onto the freshly flashed stock OS card. I am using a 64gb card. It does not matter if I flash the SD using the small image without roms or the big one with roms. in both cases it claims there is insufficient space.

**Comment by washere** [comment]
Yes it should be in homepage here, installation section, and also in wiki:
It only gave me 8gb which is not enough, SO:

in a partition manager, I am on Linux:
Open Gparted (install it) & choose SdCard (top-right dropdown menu button of devices
Choose SdCard
right-click on 8gb block & choose
RESIZE & ....
Expand to maximum (whatever your card capacity ( in popup box: drag block border or enter number)
click ok etc button at bottom
Click top toolbar APPLY button (to actually do the operation)

If on Windows or Mac, install a "Partition Manager" app to do same.

Should really say in Installation + Wiki to:
USE A PARTITION MAANGER TO EXPAND 8gb linuxrootfs PARTITION TO MAX SIZE on whatever card size.

**Comment by washere** [comment]
I think because this week i am writing to SdCard on linux (Impression app on debian). So it partitions differently.
I suggest to use rufus on Windows to create the SdCard.
Anyway i will try later today on Windows laptop which i rarely use.

**Comment by washere** [comment]
Yes, it works ok with Rufus on windows, partition auto expands on first boot.
If you see linuxrootfs of 8gb,  it did not work.
Use Rufus on windows.

On a copmuter on the SdCard: You should see /ROMS partition after first boot finishes partitioning, then was successful.
Partition "linuxrootfs" means wrong iso/img writer was used.

Installation instruction on homepage here & wiki need to specify:

* 1) supported & tested  list of SdCard image writers & which platforms, eg: Rufus on Windows etc. Balena etcher was also giving me problems on linux.
+
* 2) where /bios folder goes, ie: same root level as /Roms folder in ROMS partition.
Better yet, create an empty /bios folder there like /Roms folder is created.

**Comment by washere** [comment]
Also i think in instructions & Wiki, should say any problems with partitions or auto expanding :

"Try installing with Rufus on a Windows machine."

Some linux img writer/partitioning apps, are too nosy with doing things to partitions they think should be done. Then we get problems with auto expansion, etc.
